<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinWindow</class>
 <widget class="QMainWindow" name="JoinWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
    /*background-image: url(:/images/bg_main.jpg); */
    background-position: center;
    background-repeat: no-repeat;
    background-color: #1a1a1a;
}

QLabel {
    color: #f0f0f0;
    font-size: 11pt;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;;
}

QLineEdit {
    background-color: rgba(255, 255, 255, 0.85);
    color: #000;
    font-size: 10pt;
    padding: 5px;
    border: 1px solid #aaa;
    border-radius: 5px;
}

QPushButton {
    background-color: #4e54c8;
    background-image: linear-gradient(315deg, #4e54c8 0%, #8f94fb 74%);
    color: white;
    border-radius: 10px;
    font-size: 12pt;
    padding: 8px 16px;
    font-weight: bold;
    border: none;
}

QPushButton:hover {
    background-color: #6a6ff0;
}

QPushButton:pressed {
    background-color: #3e42a8;
}

QTableWidget {
    background-color: rgba(255, 255, 255, 0.9);
    alternate-background-color: #f2f2f2;
    gridline-color: #ccc;
    font-size: 9pt;
    color: #000;
    border-radius: 6px;
}
</string>
   </property>
   <widget class="QPushButton" name="PushButton_Back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Username">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_List">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rooms to join:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_LookForRooms">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Look For Rooms</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Username">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>someone</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Username_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Port">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6000</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label_List_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>470</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Network Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_NetworkStatus">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>540</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="TableWidget_Rooms">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>541</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
